frontends/mit-open/src/pages/SearchPage/SearchPage.tsx

```diff
@@ -0,0 +1,355 @@
+import React from "react"
+import {
+  styled,
+  Container,
+  SearchInput,
+  Pagination,
+  Card,
+  CardContent,
+  Grid,
+  Skeleton,
+} from "ol-components"
+import { MetaTags } from "ol-utilities"
+
+import { ResourceTypeEnum } from "api"
+import type { LearningResourcesSearchApiLearningResourcesSearchRetrieveRequest as LRSearchRequest } from "api"
+import { useLearningResourcesSearch } from "api/hooks/learningResources"
+
+import { GridColumn, GridContainer } from "@/components/GridLayout/GridLayout"
+import { useSearchQueryParams, FacetDisplay } from "@mitodl/course-search-utils"
+import type { FacetManifest } from "@mitodl/course-search-utils"
+import { useSearchParams } from "@mitodl/course-search-utils/react-router"
+import LearningResourceCard from "@/page-components/LearningResourceCard/LearningResourceCard"
+import PlainVerticalList from "@/components/PlainVerticalList/PlainVerticalList"
+
+import { ResourceTypeTabs } from "./ResourceTypeTabs"
+import type { TabConfig } from "./ResourceTypeTabs"
+
+const RESOURCE_FACETS: FacetManifest = [
+  {
+    name: "topic",
+    title: "Topics",
+    useFilterableFacet: true,
+    expandedOnLoad: true,
+  },
+]
+
+const AGGREGATIONS: LRSearchRequest["aggregations"] = ["resource_type", "topic"]
+
+const ColoredHeader = styled.div`
+  background-color: ${({ theme }) => theme.palette.secondary.light};
+  height: 150px;
+  display: flex;
+  align-items: center;
+`
+const SearchField = styled(SearchInput)`
+  background-color: ${({ theme }) => theme.custom.colorBackgroundLight};
+  width: 100%;
+`
+
+const FacetStyles = styled.div`
+  * {
+    color: ${({ theme }) => theme.palette.secondary.main};
+  }
+
+  input[type="checkbox"] {
+    accent-color: ${({ theme }) => theme.palette.primary.main};
+  }
+
+  .filter-section-main-title {
```

Comment:
It would be good to find a solution that doesn't depend on classnames in dependencies. I do see the dilemma - how to write `course-search-utils` that are customizable, but still largely hold their layout and base styles without having to pass so many components in that lose code reuse.

If `course-search-utils` also used styled components, we could provide them to the css template literal for interpolation (we discussed that a bit the other day). The benefit would be that `course-search-utils` API is defined by its exports and not additionally by its classnames. Not one for this PR, but food for thought.